From 1c0e3fb6deb9c3d3c1dd1d4940ffe8d71c2158c9 Mon Sep 17 00:00:00 2001
From: Bernd Waibel <waebbl@gmail.com>
Date: Mon, 20 Jan 2020 18:45:10 +0100
Subject: [PATCH] CMakeLists.txt: comment cpack.d inclusion

Directory is not included in tarball and not needed on Gentoo.

Signed-off-by: Bernd Waibel <waebbl@gmail.com>
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 775037a..e3deb01 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -627,4 +627,4 @@ install(EXPORT ${PROJECT_NAME}
 
 # ############################################################################
 # New CPACK section, please see the README file inside cpack.d directory.
-add_subdirectory(cpack.d)
+#add_subdirectory(cpack.d)
-- 
2.25.0

